states/wordpress/init.sls

```diff
@@ -133,6 +133,29 @@ include:
       - composer: {{ sls }} composer update
 
 
+{{ sls }} dir wp-content/upgrade-temp-back:
+    file.directory:
+      - name: {{ DOCROOT }}/wp-content/upgrade-temp-back
```

Comment:
the folder name here is `upgrade-temp-back` when it should be `upgrade-temp-backup`

Reply:
Good catch! I've updated it. Thank you! 👍